site/_posts/2024-12-1-fish-4b.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "fish-shell 4.0b, now in Rust"
+date: 2024-12-07
+categories: general
+published: false
+---
+
+fish is a smart and user-friendly command line shell with clever features that just work, without needing an advanced degree in bash scriptology. Today we are announcing an open beta, inviting all users to try out the upcoming 4.0 release.
+
+fish 4.0 is a big upgrade. It's got lots of new features to make using the command line easier and more enjoyable, such as more natural key binding and expanded history search. And under the hood, we’ve rebuilt the foundation in Rust to embrace modern computing[^modern-computing-advancements].
+
+🎶 The beta is open, the release notes are live, and we’re ready to party like it’s 1995! 💿
+
+### Try the fish-shell 4.0 beta at [fishshell.com](https://fishshell.com). View the [release notes here](https://fishshell.com/docs/current/relnotes.html).
```

Comment:
Do we publish the release notes here, or just on GitHub releases? They're not quite polished yet but we can mark them as a beta as well.